packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -812,6 +814,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         target={ this._comboBoxWrapper }
         onDismiss={ this._onDismiss }
         setInitialFocus={ false }
+        calloutWidth={ dropdownWidth || this._comboBoxWrapper.clientWidth + 2 }
       >
         <div className={ this._classNames.optionsContainerWrapper } ref={ this._resolveRef('_comboBoxMenu') }>
```

Comment:
There's a bit of cleanup to do here with the optionsContainerWrapper, but we'll be doing a styling sweep in the near future and we'll be able to clean it up then. So don't worry about it. This is the correct way to set the callout width.